### DISCUSSION TITLE
Process failed

### DISCUSSION THREAD

**Discussion by 28development** [spatie/browsershot#534] [Q&A]
Hi I installed this in my project(using mac). I installed using node and composer.

But doing this

```
        return Browsershot::url('http://127.0.0.1:8000/customers/infotext/1')
            ->timeout(330)
            ->save("test.pdf");
```

Gives me the following error

`The process "PATH=$PATH:/usr/local/bin NODE_PATH=`npm root -g` node '/Users/tim/Documents/code/private/office-investments/vendor/spatie/browsershot/src/../bin/browser.js' '{"url":"http:\/\/127.0.0.1:8000\/customers\/infotext\/1","action":"pdf","options":{"path":"test.pdf","args":[],"viewport":{"width":800,"height":600},"timeout":330000}}'" exceeded the timeout of 330 seconds.`

**Comment by akhan619** [comment]
Are you using blade commands in your view? Look at my answer to #516

**Comment by luciandex** [comment]
I found a possible solution on internet.
Run the following into your project root (or in workspace container if run in Docker).

$ `which node`

result ------> /usr/local/bin/node (may be different)

$ `export NODE_PATH='/usr/local/bin/node'`   <---result

$ `npm run dev` <---- probably not neccessary to rerun this(?)
